prometheus_client/parser.py

```diff
@@ -20,128 +20,96 @@ def text_string_to_metric_families(text):
         yield metric_family
 
 
-def _unescape_help(text):
-    result = []
-    slash = False
-
-    for char in text:
-        if slash:
-            if char == '\\':
-                result.append('\\')
-            elif char == 'n':
-                result.append('\n')
-            else:
-                result.append('\\' + char)
-            slash = False
-        else:
-            if char == '\\':
-                slash = True
-            else:
-                result.append(char)
+def _replace_escaping(s):
```

Comment:
Help and label values have different escaping rules (double quote is the difference), you need two functions for this.

Reply:
added another function specific for `Help`